labs/saw/arithmetic-verification/arithmetic_implementations.c

```diff
@@ -0,0 +1,198 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+
+// addition_standard
+//
+// Add two 16-bit integers in a standard way, cast each to a local 32 bit
```

Comment:
Style consistency:  "<n> bit" --> "<n>-bit" (n=32 in this case).

Reply:
Fixed.